### PR TITLE
Fix User Guide: Adding a contact/task will revert UI from filtered list back to the full list

### LINKED ISSUE #226: [PE-D][Tester D] When you add a contact while you are at the filtered list, it reverts back to full contact list
Initial State:

Filtered List with query of y

When I add a new person called Peter will be queried out as his name has no y, I get reverted back to the full contact list

This is not as ideal as I may still want to view the filtered list

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: prit3010/ped#6

### PR DESCRIPTION
**Pull request by bananamonkey4655**
Fixes #226

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -307,7 +307,8 @@ Displays all contacts matching the arguments provided by you.
 
 Format: `contact find q/QUERY`
 
-* QUERY is not case-sensitive.
+* QUERY is not case sensitive.
+* Note that adding a new contact will reset the UI back to the full list of contacts.
```
Comment (Nephelite) [inline docs/UserGuide.md:311]: Should use the ℹ️ or the 💡 notation in a `<div>` container for this note.

Also, maybe rephrase to something like: "When you add a new contact, any previous `find` command that may be in effect will be removed, returning the list of contacts to a full list."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -321,6 +322,8 @@ Format: `task find <q/QUERY x/DONE a/ASSIGNMENT>`
 * QUERY is not case-sensitive.
 * ASSIGNMENT is either FROM or TO, representing "m/" and "o/", or 'Assigned by' and 'Assigned to', respectively.
 * DONE is either X or O, representing 'Done' and 'Not done' respectively.
+* Note that adding a new task will reset the UI back to the full list of tasks.
```
Comment (Nephelite) [inline docs/UserGuide.md:325]: Refer to previous comment.

**Review by Nephelite — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (bananamonkey4655) [reply docs/UserGuide.md]: Fixed @Nephelite, added the abovementioned stuff. Lmk if needs changes

**Review by Nephelite — APPROVED** [review]
